signbank/dictionary/tasks.py

```diff
@@ -92,20 +103,13 @@ def retrieve_videos_for_glosses(video_details: List[VideoDetail]):
 
         gloss = Gloss.objects.get(pk=video["gloss_pk"])
 
-        # change video type to main for illustrations
-        video_type = validation_video_type
-        if video["title"] == "Illustration":
-            video_type = main_video_type
-            video["title"] = ""
-
         gloss_video = GlossVideo(
-            title=video["title"],
             gloss=gloss,
             dataset=gloss.dataset,
             videofile=file,
             version=video["version"],
             is_public=False,
-            video_type=video_type
+            video_type=video_type_map.get(video["video_type"], None)
```

Comment:
**question**: assuming that if we get more than 2 from the final example, it's correct to not set the type?

Reply:
I believe there is a restriction in NZSL Share about how many usage examples a user can upload, which is two. So we never should get more than two.